corehq/toggles.py

```diff
@@ -1048,6 +1048,13 @@ def _commtrackify(domain_name, toggle_is_enabled):
     )
 )
 
+APP_BUILDER_CONDITIONAL_NAMES = StaticToggle(
+    'APP_BUILDER_CONDITIONAL_NAMES',
+    'Conditional, calculation-based  mapping for menu and form names',
```

Comment:
I think generally our custom feature flag names use the project name in front of it. Like 'ICDS/REACH: Condititonal ...'

Reply:
c16d5a5f0e69b5443df44c8c3990f59cb1a8a5fa